flake.nix

```diff
@@ -471,8 +471,15 @@
             craneLib = craneLibCross.${target.attr};
             deps = craneLib.buildDepsOnly (commonArgs // {
               src = filterWorkspaceDepsBuildFiles ./.;
-              pname = "pkg-${name}-${target.attr}-deps";
-              buildPhaseCargoCommand = "cargo build --profile $CARGO_PROFILE --target ${target.name} --package ${name}";
+              pname = "fedimint-packages-${target.attr}";
+              # workaround: on wasm, we can't compile all deps, so narrow dependency build
+              # to ones used by the client package only
+              buildPhaseCargoCommand =
+                if target.name == "wasm32-unknown-unknown" then
```

Comment:
where does `wasm32-unknown-unknown` come from?

Reply:
From a cross-toolchain defintion somewhere in the file. But that's the "triple" target used for wasm. Just like `x86_64` is a common name for intel cpu architecture.